Benchmarks/Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version: 5.9
+import PackageDescription
+
+let package = Package(
+    name: "swift-openapi-runtime-benchmarks",
+    platforms: [ .macOS("14") ],
+    dependencies: [
+        .package(name: "swift-openapi-runtime", path: "../"),
+        .package(url: "https://github.com/ordo-one/package-benchmark.git", from: "1.22.0"),
+    ],
+    targets: [
+        .executableTarget(
+            name: "OpenAPIRuntimeBenchmarks",
+            dependencies: [
+                .product(name: "Benchmark", package: "package-benchmark"),
+                .product(name: "OpenAPIRuntime", package: "swift-openapi-runtime"),
+            ],
+            path: "Benchmarks/OpenAPIRuntimeBenchmarks",
```

Comment:
Do we need this nested Benchmarks folder?

Reply:
We can discuss separately.

Reply:
We don't need it. But let's see how the GHA rollout shakes out and decide if we need to align everything.